docs/sum.rst

```diff
@@ -0,0 +1,22 @@
+
+Sum Layer
+===================
+
+The sum types and functions.
```

Comment:
Please mention that this is experimental functionality and the users need to define MIOPEN_BETA_API before inclusion of miopen.h to enable it.

See also https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2481/files#r1402063767

Reply:
fixed it